kubernetes/onpremise/kubeadm/README.md

```diff
@@ -45,7 +45,7 @@ No modules.
 | <a name="input_master_node_name"></a> [master\_node\_name](#input\_master\_node\_name) | The name of the cluster master node. | `string` | `"master"` | no |
 | <a name="input_master_private_ip"></a> [master\_private\_ip](#input\_master\_private\_ip) | The private ip of the master node. | `string` | n/a | yes |
 | <a name="input_master_public_ip"></a> [master\_public\_ip](#input\_master\_public\_ip) | The public ip of the master node. | `string` | n/a | yes |
-| <a name="input_tls_private_key_pem"></a> [tls\_private\_key\_pem](#input\_tls\_private\_key\_pem) | The private key of the master node. | `string` | n/a | yes |
+| <a name="input_tls_private_key_pem_content"></a> [tls\_private\_key\_pem\_content](#input\_tls\_private\_key\_pem\_content) | The private key of the master node. | `string` | n/a | yes |
```

Comment:
Maybe you should precise in the description that it is the content of the private key and not its name ?

Reply:
yes